test/test_encryption.py

```diff
@@ -1038,38 +1081,61 @@ def setUpClass(cls):
         cls.client_encryption = ClientEncryption(
             {'aws': AWS_CREDS}, 'keyvault.datakeys', client_context.client, OPTS)
 
-    def _test_create_data_key(self, master_key):
+    def setUp(self):
+        kms_providers = {'aws': AWS_CREDS,
+                         'azure': AZURE_CREDS,
+                         'gcp': GCP_CREDS}
+        self.client_encryption = ClientEncryption(
```

Comment:
This overwrites the `cls.client_encryption` that's created in setUpClass. Suggest renaming or moving this logic to setUpClass if possible.

Reply:
The prose tests explicitly ask for recreating this object for every test so I have simply removed this from `setUpClass` - it had been left there in error.